types/logger/tokenbucket.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package logger
+
+import (
+	"time"
+)
+
+type tokenBucket struct {
+	remaining int
+	max       int
```

Comment:
can you put the immutable fields first, separated by a blank line?

Reply:
None of them are really immutable in this struct; it's meaningful to change any of them at runtime. Although we don't, in this tiny use case.